sections/appendix/taxonomies.tex

```diff
@@ -0,0 +1,87 @@
+\subsubsection{Taxonomy}\label{subsec:appendix_taxonomy}
+Taxonomy is fundamentally different from the other metadata fields, in this paper.
+It is not fully observed with only roughly $25\%$ of documents having a taxonomy field.
+It is hierarchical, with each taxonomy containing a sequence of sub-taxonomies, such as: 'STEDER/Danmark/Nordjylland/Aalborg'.
+It is also possible for each document to have multiple taxonomy sequences.
+Like with authors, we remove any sub-taxonomy that is used in less than $14$ taxonomy sequences ($0.01\%$ of the number of documents).
+Out of $1135$ sub-taxonomies, $779$ are removed during this preprocessing.
+A subset of the taxonomies and the number of articles they appear in can be seen in \autoref{tab:taxonomy_table}.
+The taxonomy tree has the following sizes: $1$, $4$, $32$, $90$, and $69.192$.
+The numbers $4$ and $32$ are based on the number of unique taxonomy fields observed in the dataset.
+The numbers $1$, $90$, and $69.192$ are the root, the number of topics, and the number of words, respectively.
```

Comment:
these sizes are based on the pachinko dag not the taxonomy tree, so they need to be updated
```suggestion
The taxonomy tree has the following sizes: $1$, $4$, $32$, $90$, and $69.192$.
The numbers $4$ and $32$ are based on the number of unique taxonomy fields observed in the dataset.
The numbers $1$, $90$, and $69.192$ are the root, the number of topics, and the number of words, respectively.
\todo{fix sizes to fit with taxonomy tree instead of pachinko dag}
```